HW7/task2.js

```diff
@@ -0,0 +1,21 @@
+// 1. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+function isPolindrom(word){
+    toLowerCaseWord = word.toLowerCase()
+    const reverseWord = toLowerCaseWord.split('').reverse().join('')
+    return result = reverseWord === toLowerCaseWord ? `${word} - It is palindrom` : `${word} - It isn't palindrom` 
```

Comment:
```suggestion
    return reverseWord === toLowerCaseWord
```
Функция должна возвращать интерпретируемый результат, и никак не отвечать за красивый вывод результата в консоль. Этим стоит заниматься вне функции